zipline/algorithm.py

```diff
@@ -1364,13 +1364,18 @@ def set_long_only(self):
     ##############
     @api_method
     @require_not_initialized(AttachPipelineAfterInitialize())
-    def attach_pipeline(self, pipeline, name):
+    def attach_pipeline(self, pipeline, name, chunk_size=None):
         """
         Register a pipeline to be computed at the start of each day.
         """
         if self._pipelines:
             raise NotImplementedError("Multiple pipelines are not supported.")
-        self._pipelines[name] = pipeline
+        if chunk_size is None:
+            # make the first chunk smaller to get more immediate results
+            chunks = iter(chain([5], repeat(126)))
```

Comment:
Should we make these symbolic constants?  I recognize 252 by sight (though even that was pushing it), but 126 isn't obviously half a year.